lib/at_host/at_host.c

```diff
@@ -61,9 +68,10 @@ static inline void write_uart_string(char *str, size_t len)
 
 static void response_handler(char *response)
 {
-	int len = strlen(response) + 1;
+	int len = strlen(response);
+
 	/* Forward the data over UART */
-	if (len > 1) {
+	if (len > 0) {
 		write_uart_string(response, len);
```

Comment:
If we are really dealing with strings here, we should probably give up on `len` parameter, and seek for NULL termination in `write_uart_string` (`while` loop instead of `for`). This should simplify logic here and there.

Reply:
Simplified as suggested, but I kept the loop, as it seemed like the easiest way to iterate through the string. Is it ok?